components/permissions/android/java_sources.gni

```diff
@@ -1,4 +1,16 @@
 brave_components_permissions_java_sources = [
+  "//brave/android/java/org/chromium/chrome/browser/crypto_wallet/KeyringServiceFactory.java",
+  "//brave/android/java/org/chromium/chrome/browser/crypto_wallet/util/Blockies.java",
+  "//brave/components/permissions/android/java/src/org/chromium/components/permissions/BraveAccountsListAdapter.java",
   "//brave/components/permissions/android/java/src/org/chromium/components/permissions/BravePermissionDialogDelegate.java",
   "//brave/components/permissions/android/java/src/org/chromium/components/permissions/BravePermissionDialogModel.java",
 ]
+
+brave_components_permissions_java_deps = [
+  "//brave/components/brave_wallet/common:mojom_java",
+  "//chrome/browser/profiles/android:java",
```

Comment:
components cannot depend on `//chrome/browser`

Reply:
gn_check passes

Reply:
that code is removed and we have a permission dialog in a browser process now